### PR TITLE
Fixes #190 Reduce the size of menu icons to half

### LINKED ISSUE #190: Reduce the size of menu icons to half
Icons are too big. Please reduce them to half the size.
screenshot from 2017-04-28 05-36-38

### PR DESCRIPTION
**Pull request by harshit98**
Resolves #190 

Preview link - <a href="https://harshit98.github.io/susper.com/">here</a>.

### CODE REVIEW

**Comment by mariobehling** [comment]
Does not work for me.

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "lint": "tslint \"src/**/*.ts\"",
     "test": "ng test",
     "pree2e": "webdriver-manager update",
-    "e2e": "protractor"
+    "e2e": "protractor",
```
Comment (isuruAb) [inline package.json]: these changes are not relevant for this issue.
can you push these changes to relevant issue ?

**Comment by isuruAb** [comment]
@harshit98  preview link is not working can you check it again

File: package.json (hunk 1 above)
Comment (harshit98) [inline package.json]: @isuruAb Actually, these changes came when I tried to deploy the branch. My cloned repo was not updated. Sorry for that.

**Comment by harshit98** [comment]
@mariobehling @isuruAb 

screenshot-from-2017-04-20-21_45_11

You must be seeing a red error button in the address bar. Click on load unsafe scripts and everything will work fine.

**Comment by mariobehling** [comment]
Not showing up. Is it maybe because it is not on susper.com?

Not working with you link.

**Comment by harshit98** [comment]
The preview link which I have provided, shows the reduced size of icons.

screenshot from 2017-05-02 09-17-10

susper.com currently shows big icons.

screenshot from 2017-05-02 09-17-26

**Comment by mariobehling** [comment]
@harshit98 Could you make the icons even smaller, please?

Currently the top right icon is aligned differently. Do you see that in your screenshot?

**Comment by harshit98** [comment]
@mariobehling Yes, I will do the changes. Just wanted to know if size of icons looks good after changes. I'll work on this and do the following changes as per you. :+1: : )

**Comment by harshit98** [comment]
Done! 

screenshot from 2017-05-04 02-13-38

**Comment by Marauderer97** [comment]
@harshit98 Looks good! One thing though, the icons look aligned to the left to me.. with slightly more space to the right. Can this be taken care of? Is this an issue with the padding percentages on the left and right?

**Comment by harshit98** [comment]
@Marauderer97 Ok, I'll look into it. It's the problem with `width` of drop-box.

**Comment by harshit98** [comment]
@Marauderer97 Done!

screenshot from 2017-05-05 18-22-18

File: package.json [hunk 2]
```diff
@@ -36,7 +36,7 @@
     "zone.js": "^0.6.23"
   },
   "devDependencies": {
-    "@angular/cli": "^1.0.0",
+    "@angular/cli": "^1.0.2",
```
Comment (Marauderer97) [inline package.json]: @harshit98 Same line of code. Though there is nothing wrong with it, I think this line needs to be in a relevant PR. We can take a second opinion on this.

**Comment by harshit98** [comment]
@Marauderer97 I cloned the fresh repo of susper yesterday, so the branches has been deleted. That is why preview link is not working. :sweat_smile:

**Pull request merged**